cmp-navigation/src/commonMain/kotlin/cmp/navigation/navigation/FeatureNavHost.kt

```diff
@@ -55,8 +58,8 @@ internal fun FeatureNavHost(
         helpNavGraph(
             findLocations = {},
             navigateBack = appState.navController::popBackStack,
-            callHelpline = {},
-            mailHelpline = {},
+            callHelpline = { callHelpline() },
```

Comment:
@revanthkumarJ look at the type of callHelpLine if it is same directly assign it. Somethin like callHelpline = callHelpline. no need to put it inside the curly braces.